File: app/src/main/java/com/example/hellooboe/ActivityTitle.java
package com.example.hellooboe;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

public class ActivityTitle extends AppCompatActivity {

    static {
        System.loadLibrary("native-lib");
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_title);
        NativeMethods.createStream();
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
        NativeMethods.destroyStream();
    }

    public void onButtonTitleNewClicked(View view) {
        startActivity(new Intent(this, ActivityMain.class));
    }

    public void onButtonTitleLoadClicked(View view) {
        startActivity(new Intent(this, ActivityLoadFile.class));
    }

    public void onButtonTitleSettingsClicked(View view) {
        startActivity(new Intent(this, ActivitySettings.class));
    }

}

File: app/src/main/java/io/fourth_finger/sound_sculptor/ActivityTitle.kt
package io.fourth_finger.sound_sculptor

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class ActivityTitle : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_title)
        NativeMethods.createStream()
    }

    override fun onDestroy() {
        super.onDestroy()
        NativeMethods.destroyStream()
    }

    fun onButtonTitleNewClicked(view: View?) {
        startActivity(Intent(this, ActivityMain::class.java))
    }

    fun onButtonTitleLoadClicked(view: View?) {
        startActivity(Intent(this, ActivityLoadFile::class.java))
    }

    fun onButtonTitleSettingsClicked(view: View?) {
        startActivity(Intent(this, ActivitySettings::class.java))
    }

    companion object {
        init {
            System.loadLibrary("sound_sculptor")
        }
    }
}